base_geoengine/geo_line.py

```diff
@@ -18,17 +18,18 @@
 #    along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #
 ##############################################################################
-from __future__ import absolute_import
-from osv import fields
+from openerp.osv import fields
 from . import geo_field
 
+
 class GeoLine(geo_field.Geom):
     """New type of column in the  ORM for POSTGIS geometry Line type"""
     _type = 'geo_line'
+    _geo_type = 'LINESTRING'
 
-    def __init__(self, string, dim=2, srid=900913 , gist_index=True, **args):
-        res = super(GeoLine, self).__init__(string, "LINESTRING", dim=dim,
-                                            srid=srid, gist_index=gist_index, **args)
-        return res
+    def __init__(self, string, dim=2, srid=900913, gist_index=True, **args):
```

Comment:
Docstring to explain the params would be nice. If it's from the super, make sure it is transfered.